bin/zkServer.sh

```diff
@@ -236,43 +223,41 @@ restart)
 status)
     # -q is necessary on some versions of linux where nc returns too quickly, and no stat result is output
     isSSL="false"
-    clientPortAddress=`$GREP "^[[:space:]]*clientPortAddress[^[:alpha:]]" "$ZOOCFG" | sed -e 's/.*=//'`
-    if ! [ $clientPortAddress ]
-    then
-	      clientPortAddress="localhost"
+    clientPortAddress=$($GREP "^[[:space:]]*clientPortAddress[^[:alpha:]]" "$ZOOCFG" | sed -e 's/.*=//')
+    if ! [ "$clientPortAddress" ]; then
```

Comment:
I think this is trying to do:

```suggestion
    if [ -z "$clientPortAddress" ]; then
```

I'm not exactly, sure, though. This is used in a few places.